integration_tests/src/test/java/io/opentelemetry/JaegerExporterIntegrationTest.java

```diff
@@ -58,7 +59,7 @@
   @SuppressWarnings("rawtypes")
   @Container
   public static GenericContainer jaegerExampleAppContainer =
-      new GenericContainer("adoptopenjdk/openjdk8")
+      new GenericContainer(DockerImageName.parse("adoptopenjdk/openjdk8"))
```

Comment:
Due to a bug in the RC of testcontainers, you need to add `:latest` here or notice the broken CI (it downloads every tag for the image ever right now...) Can you add `:lastest` to all the testcontainers untagged images?

Reply:
oh blah. ok, I'll fix it

Reply:
done. 🤞